R/mod_faithful_hist.R

```diff
@@ -3,6 +3,7 @@
 #' @description UI and server function for an example Shiny module.
 #'
 #' @param id Internal parameter for {shiny}.
+#' @param variable choices are waiting and eruptions.
```

Comment:
```suggestion
#' @param variable choices are waiting and eruptions among the faithful dataset columns
```

I believe that this would be a better docu